integrations/octopus/.port/resources/blueprints.json

```diff
@@ -0,0 +1,292 @@
+[
+  {
+    "identifier": "octopusDeployProject",
+    "title": "Octopus Deploy Project",
+    "icon": "Octopus",
+    "description": "An Octopus Deploy project",
+    "schema": {
+      "properties": {
+        "slug": {
+          "type": "string",
+          "title": "Slug",
+          "description": "The slug identifier of the project"
+        },
+        "id": {
+          "type": "string",
+          "title": "Project ID",
+          "description": "The unique identifier of the project"
+        },
+        "url": {
+          "type": "string",
+          "title": "Project URL",
+          "format": "url",
+          "description": "The URL to access the project in Octopus Deploy"
+        },
+        "description": {
+          "type": "string",
+          "title": "Description",
+          "description": "The project description"
+        },
+        "spaceId": {
+          "type": "string",
+          "title": "Space ID",
+          "description": "The ID of the space in which the project exists"
+        },
+        "isDisabled": {
+          "type": "boolean",
+          "title": "Is Disabled",
+          "description": "Indicates if the project is disabled"
+        },
+        "lifecycleId": {
+          "type": "string",
+          "title": "Lifecycle ID",
+          "description": "The lifecycle ID associated with the project"
+        },
+        "tenantedDeploymentMode": {
+          "type": "string",
+          "title": "Tenanted Deployment Mode",
+          "description": "The deployment mode regarding tenants"
+        },
+        "allowIgnoreChannelRules": {
+          "type": "boolean",
+          "title": "Allow Ignore Channel Rules",
+          "description": "Indicates if channel rules can be ignored"
+        }
+      }
+    },
+    "calculationProperties": {}
+  },
+  {
+    "identifier": "octopusDeployRelease",
+    "title": "Octopus Deploy Release",

```

Comment:
Let's remove the deploy from the title and keep it simple as Octopus Project, Octopus Deployment, Octopus Release etc

Reply:
Done